pkg/governance-scripts/contracts/BalancerContractRegistryInitializer.sol

```diff
@@ -111,11 +99,5 @@ contract BalancerContractRegistryInitializer {
         for (uint256 i = 0; i < _aliasNames.length; ++i) {
             balancerContractRegistry.addOrUpdateBalancerContractAlias(_aliasNames[i], _aliasAddresses[i]);
         }
-
-        // Renounce all roles.
-        _authorizer.renounceRole(registerContractRole, address(this));
-        _authorizer.renounceRole(addAliasRole, address(this));
```

Comment:
we should still renounce these two; I wouldn't keep them active.

Reply:
I guess it depends on their model. Would they grant, execute and revoke in one transaction? If the contract revokes them, we'd need them to know they only need to grant. I can make it do that.